RadixWallet/Features/ImportMnemonic/ImportMnemonic.swift

```diff
@@ -88,6 +88,7 @@ public struct ImportMnemonic: Sendable, FeatureReducer {
 		public struct WriteMode: Sendable, Hashable {
 			public var isProgressing: Bool
 			public let persistStrategy: PersistStrategy?
+			public let hideModeButton: Bool
```

Comment:
Well, half of the time it's the "normal mode button"...

Reply:
I actually called it that to begin with, but then I renamed it because we already refer to it as `modeButton` in the viewstate. Anyway, now I have renamed it:
<img width="403" alt="image" src="https://github.com/radixdlt/babylon-wallet-ios/assets/123396602/f040cd72-ad59-4fd9-96bc-58589de65a02">